examples/example.krun

```diff
@@ -73,3 +73,7 @@ TEMP_READ_PAUSE = 1
 
 # CPU pinning (off by default)
 #ENABLE_PINNING = False
+
+AMPERF_RATIO_BOUNDS = 0.97, 1.03
+AMPERF_BUSY_THRESHOLD = 3.4 * 1024 * 1024 * 1024  # 3.6 GHz
+OUTLIER_WINDOW_SIZE = 200
```

Comment:
This was what I was going to ask you on MM earlier. But if we kill the outliers stuff, it's moot.